roles/ceph-defaults/defaults/main.yml

```diff
@@ -463,8 +463,8 @@ ceph_tcmalloc_max_total_thread_cache: 134217728
 ##########
 # DOCKER #
 ##########
-ceph_docker_image: "ceph/daemon-base"
-ceph_docker_image_tag: latest-main
+ceph_docker_image: "ceph/ceph"
+ceph_docker_image_tag: v18
```

Comment:
the branch `main` is intended to test against the cutting edge of ceph devel which is well `latest-main`

these changes would have to be introduced directly to stable branches.

Reply:
Agree, but I don't see `latest-main` tag here: https://quay.io/repository/ceph/ceph?tab=tags